libs/content/workspace/src/show.rs

```diff
@@ -458,8 +461,31 @@ impl Workspace {
             tab_label_resp.scroll_to_me(None);
         }
 
+        // closing (just draw status icon)
+        if tab.is_closing {
+            if !self.tasks.load_or_save_queued(tab.id)
+                && !self.tasks.load_or_save_in_progress(tab.id)
+            {
+                result = Some(TabLabelResponse::Removed);
+            }
+
+            let icon_draw_pos = egui::pos2(
+                tab_label_rect.min.x + padding_x,
+                tab_label_rect.center().y - status_icon.size / 2.0,
+            );
+
+            let icon: egui::WidgetText = (&status_icon).into();
+            let icon = icon.into_galley(
+                ui,
+                Some(TextWrapMode::Extend),
+                status_icon.size,
+                egui::TextStyle::Body,
+            );
+            ui.painter()
+                .galley(icon_draw_pos, icon, ui.visuals().text_color());
+        }
```

Comment:
in `is_closing` state, tab closes itself if there are no tasks associated and draws with just the status icon in the tab strip